cmd/aerial/pvfm.go

```diff
@@ -47,16 +47,18 @@ func np(s *discordgo.Session, m *discordgo.Message, parv []string) error {
 
 		return nil
 	} else {
+		result = append(result, ":radio: *Now Playing on PVFM*\n")
```

Comment:
Unfortunately, discord's API requires that emojis like that need to be raw byte for byte put into the string. Go supports emoji in strings though.